services/credential-server/src/agent.ts

```diff
@@ -37,8 +39,32 @@ class Agent {
   }
 
   async start(): Promise<void> {
-    await this.signifyApi.start();
-    await this.signifyApiIssuer.start();
+    await signifyReady();
+    let bran;
+    let issuerBran;
+    const bransFilePath = "../brans.json";
+    if (!existsSync(bransFilePath)) {
+      // Create new file if the file doesn't exist
+      await writeFile(bransFilePath, "");
+    }
+    const bransFileContent = await readFile("../brans.json", "utf8");
```

Comment:
We need to update the docker volume to persist this in a new volume, so it's kept on restart. So really it should be stored in a different location than with the source code